### PR TITLE
Add "others" section to the x-chain chart

### PR DESCRIPTION
**Pull request by valentinoConti**
### Changes Done

- Implement the "others" section on the sankey chart that adds all the volumes and percentages of the chains that are not top 9
- There was a consts.ts and a conts.ts file, unified them

### Screenshots

desk
mob

Implements #89

### CODE REVIEW

File: src/consts.ts [hunk 1]
```diff
@@ -1,6 +1,123 @@
...
+    name: "FTX",
+  };
+
+  const v3 = v2;
+  v3[7] = {
+    pubkey: "0x15e7cAF07C4e3DC8e7C469f92C8Cd88FB8005a20",
+    name: "xLabs",
+  };
+
+  if (version === 1) {
```
Comment (sebastianscatularo) [inline src/consts.ts:102]: probably a dictionary will work better here, something like:

```js
const pubKeys = {
  1: v1,
  2: v2,
  3: v3
}

return pubKeys[version] || []
```

File: src/components/molecules/CrossChainChart/Chart.tsx [hunk 2]
```diff
@@ -239,3 +239,59 @@ export const Chart = ({ data, selectedType }: Props) => {
     </>
   );
 };
+
+const OTHERS_FAKE_CHAIN_ID = 123123123 as ChainId;
```
Comment (sebastianscatularo) [inline src/components/molecules/CrossChainChart/Chart.tsx:243]: why do you need this fake chain id, which could be replaced by 0x0? for example?

**Review by sebastianscatularo — APPROVED** [review]

File: src/components/molecules/CrossChainChart/Chart.tsx (hunk 2 above)
Comment (valentinoConti) [reply src/components/molecules/CrossChainChart/Chart.tsx:243]: When creating a "Others" tab for the sankey, it has the same structure than a regular tab which contains info for a specific blockchain; so I need to put a `chainId` property on the tab that represents "Other" (only for this chart, not anywhere else on the app), so I assign it a random number that it's not a blockchainId and will never be, to know that tab is the "Others" tab (If I replace it for 0x0 it goes "Unset")

Don't know if I was clear 🙏🏼

File: src/consts.ts (hunk 1 above)
Comment (valentinoConti) [reply src/consts.ts]: Done ✅

**Pull request merged**